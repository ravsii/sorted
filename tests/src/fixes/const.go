package tests

const A1 = 0

const (
	A2 = iota
	B2
)

// const (
// 	B3 = iota
// 	A3
// )

// const (
// 	A4 = iota
// 	B4
//
// 	B5
// 	A5
// )

const (
	A6, B6, C6 = 0, 0, 0
	C7, B7, A7 = 3, 2, 1
)

const A8, B8, C8 = 0, 0, 0

const C9, B9, A9 = 3, 2, 1
