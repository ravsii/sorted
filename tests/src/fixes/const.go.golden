package tests

const A1 = 0

const (
	A2 = iota
	B2
)

// const (
// 	B3 = iota
// 	A3
// )

// const (
// 	A4 = iota
// 	B4
//
// 	B5
// 	A5
// )

const (
	A6, B6, C6 = 0, 0, 0
	A7, B7, C7 = 1, 2, 3
)

const A8, B8, C8 = 0, 0, 0

const A9, B9, C9 = 1, 2, 3
